.golangci.yml

```diff
@@ -0,0 +1,29 @@
+# Options for analysis running.
+run:
+  # vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
```

Comment:
recommends to add 
``` yaml  
# Include test files or not.
# Default: true
tests: false
```

<img width="714" alt="image" src="https://user-images.githubusercontent.com/43714056/212587593-38b947ef-dc4c-4f1e-8986-df908f300438.png">